spacy/tests/pipeline/test_coordinationruler.py

```diff
@@ -0,0 +1,166 @@
+import pytest
+from typing import List
+
+from spacy.tokens import Doc
+import spacy
+
+from spacy.pipeline.coordinationruler import split_noun_coordination
+
+@pytest.fixture
+def nlp():
+    return spacy.blank("en")
+
+### NOUN CONSTRUCTION CASES ###
```

Comment:
and then this is how i'm testing it - manually defining pos_tags, dep_relations and relationships between toks for different construction cases